RichText.cpp

```diff
@@ -32,6 +67,48 @@ void RichText::Line::setFont(const sf::Font &font)
 }
 
 
+////////////////////////////////////////////////////////////////////////////////
+std::size_t RichText::Line::getLength() const
+{
+    std::size_t count = 0;
+    for(sf::Text &text : m_texts)
+    {
+        count += text.getString().getSize();
+    }
+    return count;
+}
+
+
+////////////////////////////////////////////////////////////////////////////////
+sf::Color RichText::Line::getCharacterColor(std::size_t pos) const
+{
+    assert(pos < getLength());
+    assert(pos >= 0);
```

Comment:
Pointless as std::size_t is guaranteed to be unsigned, so it can never be smaller than 0.